### DISCUSSION TITLE
Couldn't find save state

### DISCUSSION THREAD

**Discussion by al2meta** [webrcade/webrcade#172] [General]
Hi guys. First of all, I would like to say that WebRCade is a great online emulator. Awesome job raz0red. The only problem I have is that I couldn't see the save state option when I pause. According to the version log, it now supports Save State so I'm not sure what I'm doing wrong. I also read the help file on the website, The picture shown on the website does not match the standalone app I'm trying to embed. I'm only seeing View Controls, SNES Settings, and Resume. Please help...

**Comment by raz0red** [comment]
Thanks for the kind words :-)If you enable cloud saves in settings, you should see the save state options.Thanks,Chris.On May 7, 2024, at 10:48 PM, AL2Meta ***@***.***> wrote:﻿
Hi guys. First of all, I would like to say that WebRCade is a great online emulator. Awesome job raz0red. The only problem I have is that I couldn't see the save state option when I pause. According to the version log, it now supports Save State so I'm not sure what I'm doing wrong. I also read the help file on the website, The picture shown on the website does not match the standalone app I'm trying to embed. Please help...
